docs/architecture/dapr-for-net-developers/getting-started.md

```diff
@@ -371,34 +371,33 @@ In the final part of this example, you'll add container support and run the solu
     The *Dockerfile* contains the YAML:
 
     ```yml
-    FROM mcr.microsoft.com/dotnet/aspnet:3.1 AS base
+    FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
     WORKDIR /app
     EXPOSE 80
-    EXPOSE 443
```

Comment:
Leave the 443 port in for the front-end. If you follow the instructions closely, you only disable HTTPS for the back-end project. The Docker file that VS generates for the front-end will contain both 80/443 ports.